frontend/src/metabase/hooks/use-scroll-on-mount.js

```diff
@@ -5,7 +5,7 @@ export const useScrollOnMount = () => {
 
   useEffect(() => {
     if (ref.current) {
-      ref.current.scrollIntoView({ block: "center" });
+      ref.current.scrollIntoView?.({ block: "center" });
```

Comment:
this change was necessary to make unit tests pass, I don't think jest-dom knows about scrolling into view, so it kept throwing "this is not a function"